requirements-dev.txt

```diff
@@ -2,3 +2,5 @@ mock==1.0.1
 httmock==1.2.2
 flake8-quotes==0.2.4
 Flask==0.10.1
+click==6.6
+tabulate==0.7.5
```

Comment:
I we are added an entry point for the command line tool, these should probably be in requirements.txt?